app/models/manageiq/providers/amazon/inventory/collector/cloud_manager.rb

```diff
@@ -4,7 +4,7 @@ def instances
   end
 
   def flavors
-    ManageIQ::Providers::Amazon::InstanceTypes.all
+    hash_collection.new(aws_ec2.client.describe_instance_types[:instance_types])
```

Comment:
Any filters we should apply here? Do we want to cap the results? Just checking:

https://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Client.html#describe_instance_types-instance_method

Reply:
No need to cap the results, but we do need to handle paging generally.  That is out of scope of this PR since that is a general issue with all of these API calls.  I opened an issue to track that, https://github.com/ManageIQ/manageiq-providers-amazon/issues/583

Reply:
So then it just becomes `hash_collection.new(aws_ec2.client.describe_instance_types.flat_map(&:instance_types))`